bindings/node/lib/clientEncryption.js

```diff
@@ -589,44 +610,29 @@ module.exports = function (modules) {
      * }
      */
     encrypt(value, options, callback) {
-      const bson = this._bson;
-      const valueBuffer = bson.serialize({ v: value });
-      const contextOptions = Object.assign({}, options);
-      if (options.keyId) {
-        contextOptions.keyId = options.keyId.buffer;
-      }
-      if (options.keyAltName) {
-        const keyAltName = options.keyAltName;
-        if (options.keyId) {
-          throw new TypeError(`"options" cannot contain both "keyId" and "keyAltName"`);
-        }
-        const keyAltNameType = typeof keyAltName;
-        if (keyAltNameType !== 'string') {
-          throw new TypeError(
-            `"options.keyAltName" must be of type string, but was of type ${keyAltNameType}`
-          );
-        }
-
-        contextOptions.keyAltName = bson.serialize({ keyAltName });
-      }
-
-      const stateMachine = new StateMachine({
-        bson,
-        proxyOptions: this._proxyOptions,
-        tlsOptions: this._tlsOptions
-      });
-      const context = this._mongoCrypt.makeExplicitEncryptionContext(valueBuffer, contextOptions);
-
-      return promiseOrCallback(callback, cb => {
-        stateMachine.execute(this, context, (err, result) => {
-          if (err) {
-            cb(err, null);
-            return;
-          }
+      return maybeCallback(() => this._encrypt(value, false, options), callback);
+    }
 
-          cb(null, result.v);
-        });
-      });
+    /**
+     * Encrypts a Match Expression or Aggregate Expression to query a range index.
+     *
+     * Only supported when queryType is "rangePreview" and algorithm is "RangePreview".
+     *
+     * @experimental The Range algorithm is experimental only. It is not intended for public use. It is subject to breaking changes.The aggregation or match expression you wish to encrypt.  The value must be in the form
```

Comment:
```suggestion
     * @experimental The Range algorithm is experimental only. It is not intended for public use. It is subject to breaking changes. The value must be in the form
```

Or should this really be part of the docs here?